test_hint_command.md

```diff
@@ -0,0 +1 @@
+go test -timeout 30s -run ^TestSendRaw$ github.com/iotexproject/iotex-core/ioctl/newcmd/action
```

Comment:
Delete temp file for not used in product env.
and directly putting in main dir  is not recommended.

Reply:
I didn't notice that I've commited this file, thanks for the advice!